website/docs/upgrades/intro.md

```diff
@@ -0,0 +1,29 @@
+---
+id: intro
+title: Introduction
+sidebar_label: Introduction
+---
+
+Upgrading to a new Hydra version is usually an easy process.
+
+## Major version upgrades
```

Comment:
Hydra uses an unconventional definition of what is "major" vs "minor" versions => it may be worth clarifying what is meant by "major" here.

Reply:
Yes. I am wondering if I want to switch to a more traditional semver, but in truth I am only using two version types now, major and patch.